### PR TITLE
Security Onion configuration instructions

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description

Security Onion integration instructions in PacketFence administration guide
# Impacts

None! It's doc !
# Dependencies
#781
# Delete branch after merge

YES

### CODE REVIEW

**Comment by louismunro** [comment]
Ready for review?

**Comment by dwlfrth** [comment]
I'll change the status once it will be ready for review ;)

**Comment by louismunro** [comment]
ok, just checking in case you had forgotten.
I'll go bugger off Zammit instead...

**Comment by dwlfrth** [comment]
Review me

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -2218,6 +2218,64 @@ Configuration
...
+Since a security suite consists of multiple pieces of software tied together, you may be prompted for different options during the installation process. A detailled "Production Deployment" guide can also be found directly from the Security Onion website: https://github.com/Security-Onion-Solutions/security-onion/wiki/ProductionDeployment
+
+PacketFence integration
++++++++++++++++++++++++
+
+Once Security Onion is installed and basically configured, integration with PacketFence is required to be able to raise violations based on sensor(s) alerts. syslog is used to forward sensor(s) alerts from Security Onion to the PacketFence detection mecanisms.
+
+The most simple way is as follow (based on https://github.com/Security-Onion-Solutions/security-onion/wiki/ThirdPartyIntegration);
+
+On the Security Onion server;
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: :

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -2218,6 +2218,64 @@ Configuration
...
+      source(s_sguil);
+      filter(f_sguil);
+      destination(d_sguil_udp);
+  };
+
+A restart of the syslog-ng daemon is then required
+
+  service syslog-ng restart
+
+On the PacketFence server;
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: :

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 3]
```diff
@@ -2218,6 +2218,64 @@ Configuration
 
 PacketFence will provide you with a basic `suricata.yaml` that you can modify to suit you own needs.  The file is located in `/usr/local/pf/conf`.
 
+Security Onion
+^^^^^^^^^^^^^^
+
+Installation and Configuration
+++++++++++++++++++++++++++++++
+
+Security Onion is a Ubuntu based security suite. Latest installation instructions are available directly from the Security Onion website, https://github.com/Security-Onion-Solutions/security-onion/wiki/Installation
```
Comment (louismunro) [inline docs/PacketFence_Administration_Guide.asciidoc]: The latest

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 4]
```diff
@@ -2218,6 +2218,64 @@ Configuration
...
+
+Since a security suite consists of multiple pieces of software tied together, you may be prompted for different options during the installation process. A detailled "Production Deployment" guide can also be found directly from the Security Onion website: https://github.com/Security-Onion-Solutions/security-onion/wiki/ProductionDeployment
```
Comment (louismunro) [inline docs/PacketFence_Administration_Guide.asciidoc]: detailed (one L)

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 5]
```diff
@@ -2218,6 +2218,64 @@ Configuration
...
+
+PacketFence integration
++++++++++++++++++++++++
+
+Once Security Onion is installed and basically configured, integration with PacketFence is required to be able to raise violations based on sensor(s) alerts. syslog is used to forward sensor(s) alerts from Security Onion to the PacketFence detection mecanisms.
```
Comment (louismunro) [inline docs/PacketFence_Administration_Guide.asciidoc]: s/basically/minimally/ ?

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 6]
```diff
@@ -2218,6 +2218,64 @@ Configuration
...
+
+The most simple way is as follow (based on https://github.com/Security-Onion-Solutions/security-onion/wiki/ThirdPartyIntegration);
```
Comment (louismunro) [inline docs/PacketFence_Administration_Guide.asciidoc]: The simplest.

**Comment by louismunro** [comment]
Minor bitching about wording. Merge away otherwise.

**Pull request merged**